src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -356,20 +357,41 @@ void manualRegistrationToolBox::computeRegistration()
 
 void manualRegistrationToolBox::reset()
 {
+    medAbstractImageView * viewFuse   = qobject_cast<medAbstractImageView*>(d->bottomContainer->view());
+    medAbstractImageView * viewMoving = qobject_cast<medAbstractImageView*>(d->rightContainer->view());
+    medAbstractImageView * viewFix    = qobject_cast<medAbstractImageView*>(d->leftContainer->view());
+
     if (d->controller)
     {
         d->controller->Reset(); // Delete every landmark on both view
+
+        // Refresh views except at closing
+        if (viewMoving)
+        {
+            d->rightContainer->view()->viewWidget()->update();
```

Comment:
You should call this directly on viewMoving since you already have a pointer to it. Same thing for viewFix below.